resources/sdk/purecloudjava/templates/README.mustache

```diff
@@ -168,22 +168,121 @@ Specify the connector in the builder:
 #### Setting the max retry time
 
 By default, the Java SDK does not automatically retry any failed requests.
-To enable automatic retries, provide a RetryConfiguration object with the maximum number of seconds to retry requests when building the ApiClient instance.
+To enable automatic retries, provide a RetryConfiguration object with the maximum number of seconds to retry requests and the max number of retries when building the ApiClient instance.
 
 Building a `RetryConfiguration` instance:  
 ```{"language":"java"}
 ApiClient.RetryConfiguration retryConfiguration = new ApiClient.RetryConfiguration();
 retryConfiguration.setMaxRetryTimeSec(30);
+retryConfiguration.setRetryMax(5);
 ```
 
 Setting `RetryConfiguration` instance to `ApiClient`:
 ```{"language":"java"}
         .withRetryConfiguration(retryConfiguration)
 ```
 Set the `maxRetryTimeSec` to the number of seconds to process retries before returning an error.
+Set the 'MaxRetryTimeSec' to the number of seconds to process retries before returning an error.
 When the retry time is a a positive integer, the SDK will follow the recommended backoff logic using the provided configuration.
 The best practices are documented in the [Rate Limiting](https://developer.mypurecloud.com/api/rest/rate_limits.html) Developer Center article.
 
+#### SDK Logging
+
+Logging of API requests and responses can be controlled by a number of parameters on the `Configuration`'s `Logger` instance.
+
+`LogLevel` values:
+1. trace (HTTP Method, URL, Request Body, HTTP Status Code, Request Headers, Response Headers)
+2. debug (HTTP Method, URL, Request Body, HTTP Status Code, Request Headers)
+3. error (HTTP Method, URL, Request Body, Response Body, HTTP Status Code, Request Headers, Response Headers)
+4. none - default
+
+`LogFormat` values:
+1. JSON
+2. Text - default
+
+By default, the request and response bodies are not logged because these can contain PII. Be mindful of this data if choosing to log it.  
+To log to a file, provide a `LogFilePath` value. SDK users are responsible for the rotation of the log file.
+
+Example logging configuration:
+```{"language":"java"}
+ApiClient.LoggingConfiguration loggingConfiguration = new ApiClient.LoggingConfiguration();
+loggingConfiguration.setLogLevel("trace");
+loggingConfiguration.setLogFormat("json");
+loggingConfiguration.setLogRequestBody(true);
+loggingConfiguration.setLogResponseBody(true);
+loggingConfiguration.setLogToConsole(true);
+loggingConfiguration.setLogFilePath("/var/log/javasdk.log");
+```
```

Comment:
Show that the APIClient must be built using `withLoggingConfiguration(loggingConfiguration)` to apply it